datajoint_utilities/dj_search/lists.py

```diff
@@ -75,18 +74,20 @@ def drop_schemas(prefix, dry_run=True, ordered=False, force_drop=False):
 
     elif not dry_run:
         while schemas_with_prefix:
+            recent_errs = []  # Refresh recent_errs at start of while loop
```

Comment:
uninformative comment
```suggestion
            recent_errs = []
```